packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -1967,6 +1968,7 @@ export class StripeHelper {
    *   - No email on the customer object.
    */
   async extractInvoiceDetailsForEmail(invoice: Stripe.Invoice) {
+    console.log('Reino ----- Email should get here right?');
```

Comment:
This should be removed before the machines learn who we are and what we want

Reply:
Lol yikes. All console.logs should be removed now.